### PR TITLE
Add LetsGrief.net again

### PR DESCRIPTION
**Pull request by InvalidTeslaa**
## Type of change

- [x] Add new directory for a server
- [x] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/letsgrief/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "server_name": "letsgrief",
+    "nice_name": "LetsGrief",
+    "direct_ip": "mc.letsgrief.net",
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "social": {
+        "web": "https://web.letsgrief.net",
```
Comment (maaaathis) [inline minecraft_servers/letsgrief/manifest.json:10]: Website is unavailable

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/letsgrief/manifest.json (hunk 1 above)
Comment (InvalidTeslaa) [reply minecraft_servers/letsgrief/manifest.json]: Fixed, was the wrong url

File: minecraft_servers/letsgrief/manifest.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "nice_name": "LetsGrief",
+    "direct_ip": "mc.letsgrief.net",
+    "supported_languages": [
+        "en",
+        "de"
+    ],
+    "social": {
+        "web": "https://letsgrief.net",
+        "web_shop": "https://store.letsgrief.net",
+        "web_support": "https://letsgrief.net/support",
```
Comment (DoJapHD) [inline minecraft_servers/letsgrief/manifest.json:12]: Can't be behind login screen

**Review by DoJapHD — CHANGES_REQUESTED** [review]

File: minecraft_servers/letsgrief/manifest.json (hunk 2 above)
Comment (InvalidTeslaa) [reply minecraft_servers/letsgrief/manifest.json]: Removed

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**